linalg/hypre.hpp

```diff
@@ -549,6 +565,11 @@ class HypreParMatrix : public Operator
    Type GetType() const { return Hypre_ParCSR; }
 };
 
+#if MFEM_HYPRE_VERSION >= 21800
+int BlockInvScal(const HypreParMatrix *A, HypreParMatrix *C,
```

Comment:
This is a matter of opinion, but I think it would be better to name this something like BlockInverseScale (no need to abbreviate quite so much). Once names get set, they are permanent for backward compatibility, so we should be careful about naming.